Source/WebKit/Shared/RemoteLayerTree/RemoteLayerTreePropertyApplier.mm

```diff
@@ -264,8 +264,14 @@ static void updateCustomAppearance(CALayer *layer, GraphicsLayer::CustomAppearan
     if (properties.changedProperties & LayerChange::FiltersChanged)
         PlatformCAFilters::setFiltersOnLayer(layer, properties.filters ? *properties.filters : FilterOperations());
 
-    if (properties.changedProperties & LayerChange::AnimationsChanged)
+    if (properties.changedProperties & LayerChange::AnimationsChanged) {
+#if ENABLE(THREADED_ANIMATION_RESOLUTION)
+        LOG_WITH_STREAM(Animations, stream << "RemoteLayerTreePropertyApplier::applyProperties() found " << properties.animationChanges.effects.size() << " effects");
+        layerTreeNode->setAcceleratedEffectsAndBaseValues(properties.animationChanges.effects, properties.animationChanges.baseValues, layerTreeHost);
+#else
```

Comment:
Can you explain, for my own edification, why you are moving this code earlier?

Reply:
This is just where the non-animation-thread code for receiving animation changes (to pass them to CA, when UI-side compositing is enabled) was. I thought having the two paths for handling a `LayerChange::AnimationsChanged` message in the same place made sense, the specific place probably doesn't matter.

It does however probably need a settings check, as well as the compile time define.